### PR TITLE
Added confirm dialog box for deleting SP's

### PR DESCRIPTION
**Pull request by nickbristow**

### CODE REVIEW

**Comment by jessieay** [comment]
is there a ticket for this? If not, let's make one. I think we should make sure there are GH tickets to track all of our work for the dashboard.

File: config/locales/en.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ en:
       edit_service_provider: Edit
       trigger_idp_refresh: Publish service providers
     required_field: Indicates a required field.
+    confirm: 'Are you sure?'
```
Comment (jessieay) [inline config/locales/en.yml]: maybe "Are you sure you want to delete this service provider? This cannot be undone."

**Comment by nickbristow** [comment]
@jessieay yep. https://github.com/18F/identity-private/issues/1707

**Comment by jessieay** [comment]
Cool, I guess it didn't link back here (which is how I usually look for it) because you linked to the PR with the `/files` dir instead of the PR itself. Weird!

**Pull request merged**